huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/Dockerfile.gpu

```diff
@@ -223,8 +223,6 @@ RUN HOME_DIR=/root && \
 
 # From Scallion open-source dependencies
 COPY llm-hosting-container/huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/THIRD-PARTY-LICENSES /root/THIRD-PARTY-LICENSES
-# From generate_oss_compliance.sh
-COPY llm-hosting-container/huggingface/pytorch/tgi/docker/1.0.3/py3/cu118/PYTHON_PACKAGES_LICENSES /root/PYTHON_PACKAGES_LICENSES
```

Comment:
Why is this needed? I.e. Why is this change to 1.0.3 needed?

Reply:
I just found out that this line https://github.com/awslabs/llm-hosting-container/pull/34/files#diff-99e58afe5cc09ca032494c7b0d4aad4a4439e65c478df4d5a778dd5c1bcf350fL221 already generated this python dependency license file in the image. So we don't need to import manually here. Thanks

Reply:
Even with that being the case, I am wary of modifying past image versions. Will it be sufficient to just omit this line in versions >=1.1.0 ?